src/screens/Likes.jsx

```diff
@@ -1,19 +1,23 @@
 import React, { useEffect } from "react";
 import styles from "./Likes.module.css";
 
-import { fetchLikes } from "../utils/request";
+import { fetchLikes, getMatches } from "../utils/request";
 import { useState } from "react";
 import Loader from "../components/Loader";
 import Swiper from "../components/Swiper";
 import Match from "../components/Match";
 
 const Likes = () => {
   const [likes, setLikes] = useState(null);
+  const [match, isMatch] = useState(null);
```

Comment:
A little comment, you should always name your state setter with a verb. 

In this case, the state variable would be `isMatch` and the setter would be `setIsMatch`. This way you avoid confusion.